yarn.lock

```diff
@@ -7075,7 +7075,7 @@ emotion-theming@^10.0.7:
     hoist-non-react-statics "^3.3.0"
     object-assign "^4.1.1"
 
-emotion@^10.0.6:
+emotion@^10.0.6, emotion@^10.0.9:
```

Comment:
Just checking we want these multiple versions

Reply:
Good catch. Looks like we've got 10.0.6 in some packages and 10.0.9 in another. Made a follow-up issue to document #526